index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link href="https://fonts.googleapis.com/css?family=Poppins:400,600" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <Header>
+        <div class="logo">My Logo</div>
+        <div class="menu"><img src="images/Union.png" alt="menu"></div>
+    </Header>
+    <main>
+        <article>
+            <h1>I’m Vishal <br>Glad to see <br>you!</h1>
+            <p>I’m a software developer! I can help you build a product, feature or website Look through some of my
+                work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+                contact me.</p>
+            <h2>LET’S CONNECT</h2>
+            <div class="social1"><img src="images/twitter.png" alt="twitter"></div>
+            <div class="social"><img src="images/Linkedin.png" alt="Linkedin"> </div>
+            <div class="social"><img src="images/medium.png" alt="medium"></div>
+            <div class="social"> <img src="images/github.png" alt="github"></div>
+            <div class="social"><img src="images/angellist.png" alt="angellist"></div>
```

Comment:
- [ ] The list should have a `ul` tag and `li` tag to wrap and they should have all an `a` see the  screenshot 👇
![li](https://user-images.githubusercontent.com/61562626/129187821-d8697c53-717b-4259-85df-e0a54d5ee077.jpeg)